index.js

```diff
@@ -1322,6 +1328,9 @@ var BP = Buffer.prototype
 
 /**
  * Augment a Uint8Array *instance* (not the Uint8Array class!) with Buffer methods
+ *
+ * This function is also used externally by `typedarray-to-buffer`.
+ *
  */
 Buffer._augment = function _augment (arr) {
   arr.constructor = Buffer
```

Comment:
Thinking about it a bit more, I think we actually need this line. Not for this package, but for `typedarray-to-buffer`. (This doesn't apply if you make the change you mention below about switching to setting the prototype.)